task_4.py

```diff
@@ -0,0 +1,5 @@
+#4
+vegetarians = ["Serhiy", "Anastasiia", "Olena", "Oksana", "Maksym", "Nadiia"]
+omnivores = ["Stepan", "Vasyl", "Solomiia", "Polina", "Anhelina"]
+omnivores.extend(vegetarians)
+print(vegetarians)
```

Comment:
you change the original omnivores data. it is not a good practice.
and no empty line